docs/pages/guides/send-event-in-server.md

```diff
@@ -0,0 +1,111 @@
+			eventBuilder.WithSeverity(events.Error).WithDescription(fmt.Sprintf("Encountered an issue while retrieving kubernetes context for connection ID `%s`", connectionID)).
+				WithMetadata(map[string]interface{}{
+					"error":                 err,
+					"probable_cause":        "Meshery might not be able to access Kubeconfig.",
+					"suggested_remediation": "Please ensure that Mehsery can establish connectivity with kubernetes.",
+				})
+			event := eventBuilder.Build()
+		    _ = provider.PersistEvent(event)
+		    go h.config.EventBroadcaster.Publish(userID, event)
+			w.WriteHeader(http.StatusInternalServerError)
+			fmt.Fprintf(w, "failed to get kubernetes context for the given ID")
+			return
+		}
+		eventBuilder.WithSeverity(events.Success).WithDescription("Retrieved kubernetes context successfully").
+			WithMetadata(map[string]interface{}{
+		 		"context_name": k8sContext.Name,
+				"server":       k8sContext.Server,
+			})
+		    event := eventBuilder.Build()
+		    _ = provider.PersistEvent(event)
+		    go h.config.EventBroadcaster.Publish(userID, event)
+
+		// Remaining code ...
+    }
+}
+ </div>
+</pre>
+&nbsp;&nbsp;
+
+
+In the given code block, the "resource" refers to the connection on which the server is performing the "ping" action. The "ActedUpon" field is supplied with the "connectionID" value obtained from the query parameter. `provider.GetK8sContext` returns the [K8sContext](https://github.com/meshery/meshery/blob/995ab671a12013088f874430cfa2c0f025b073d2/server/models/k8s_context.go#L26) object which serves as a representation of a Kubernetes context within the Meshery. if `provider.GetK8sContext` returns an error, the event is constructed with severity [Error](https://github.com/meshery/meshkit/blob/ea3c60907a1cd1902902a4113206579992772083/models/events/events.go#L18), description and enriched with metadata fields to provide the user with a complete context about the event.
+
+If `provider.GetK8sContext` succeeds, an event with a [Success](https://github.com/meshery/meshkit/blob/ea3c60907a1cd1902902a4113206579992772083/models/events/events.go#L21) severity level is constructed. The event includes a success message and metadata containing the context name and the address of the Kubernetes API server, obtained from the operation. This information aims to provide users with relevant details about the successful operation.
+
+after constructing the Event object using the Build function, event is stored into database for records and subsequently published in a new goroutine through [EventBroadcaster](https://github.com/meshery/meshery/blob/995ab671a12013088f874430cfa2c0f025b073d2/server/models/handlers.go#L241) member variable of [HandlerConfig](https://github.com/meshery/meshery/blob/995ab671a12013088f874430cfa2c0f025b073d2/server/models/handlers.go#L206), which is part of [Handler](https://github.com/meshery/meshery/blob/995ab671a12013088f874430cfa2c0f025b073d2/server/handlers/handler_instance.go#L18) instance.
+
```

Comment:
@ptdatta have you documented the Notification Formatters, yet? A link and description of them needs to be included here.

Reply:
No Currently I am working on the meshmodel UI. OK I'll do it by today.

Reply:
@leecalcote  Done✅ [#9086](https://github.com/meshery/meshery/pull/9086)

Reply:
Thanks, @ptdatta